Doc/library/sys.rst

```diff
@@ -1172,6 +1172,14 @@ always available.
    implement a dynamic prompt.
 
 
+.. data:: python_libdir
```

Comment:
The reason for that is that `libdir` is an autoconf variable: https://www.gnu.org/prep/standards/html_node/Directory-Variables.html

Reply:
I hesitate between python_libdir and libdir. I'm fine with sys.libdir: sys.prefix, sys.executable, etc. don't have "python_" prefix.

But yes, the "python" prefix is to avoid conflict with autotools standard --libdir option of ./configure script: this option has a different meaning.

Reply:
@stratakis  sure, but the `sys` module isn't autoconf-specific 😉 .

Reply:
I saw `libdir_name` in a previous commit, that sounds like a clear name to me!